### ISSUE TITLE
[2022 spring] NeurIPS 2021 The Emergence of Objectness: Learning Zero-shot Segmentation from Videos (20226056)

### ISSUE THREAD

**Issue by emjay73** [awesome-davian/awesome-reviews-kaist#435]
- Reference section is empty
- In the Method section, I somehow understand the proposed method, but I still do not understand what will be the final output of the proposed network. Will it be all 5 segmentation channels?
- I wonder what happens to the number of segments when the video contains many moving objects heading towards different directions.
- How is it possible to detect salient objects in static images using the proposed method?

**Comment by ddehun** [comment]
@emjay73 안녕하세요! 혹시 해당 리뷰의 작성자가 아니라 피어리뷰어시면 댓글로 피어 리뷰를 부탁드립니다! 이슈 제목의 학번이 리뷰 article의 작성자 분의 학번이 아니라면 이 부분도 수정 부탁드립니다 ㅎㅎ

**Comment by emjay73** [comment]
@ddehun 기입된 학번은 저의 것은 아니고 리뷰 작성자의 학번을 엑셀에 찾아 기입한 것입니다.
댓글로 달아야한다는 말씀은 위에 작성된 것을 지우고 지금 쓰고있는 글 처럼 comment를 달라는 말씀이 맞을까요?

**Comment by ddehun** [comment]
@emjay73 아하 넵넵 위에 있는 내용을 지우고 comment로 남겨주시면 될 것 같습니다!! 감사합니다

**Comment by ChoiDM** [comment]
Thank you for sharing your insightful review.

There are several points that I noticed:
- References are omitted
- Replacing the equations to the center position would be better
- Increasing the fontsize of subsection title (e.g., Related work, Idea) would be better
- It would be better to make the font of titles in 3. Method section (e.g., Appearance pathway for feature-based segmentation) bold and remove the repeated numbering (all subsection have the first 1. numbering)

**Comment by Anderlaxe** [comment]
Very interesting review. I notices some typos

- Subsections of section 3 all have a head number of 1 instead of something like 3.1, 3.2 etc
- At the very end of section 3, the variables X_i and  X_j are not written in an equation format which can confuse the reader.
- In the description of the experimental setup (section 4) the values of the learning rate and weight decay are missing.

There is no reference in the paper. It would be interesting to mention the articles used in the description of the related work.

The issues and main interests of this article are clearly defined. The content is concise and points to the essential parts of the paper.
